drivers/pwm/Kconfig

```diff
@@ -14,6 +14,12 @@ module = PWM
 module-str = pwm
 source "subsys/logging/Kconfig.template.log_config"
 
+config PWM_INIT_PRIORITY
+	int "PWM initialization priority"
+	default 70
+	help
+	  System initialization priority for PWM drivers.
```

Comment:
this needs to go into a separate commit (or PR) and all drivers should be updated accordingly.

Reply:
I've updated this in a separate commit

Reply:
@gmarull I've moved it into it's own PR now.